openstf/feature_engineering/feature_applicator.py

```diff
@@ -66,6 +64,9 @@ def add_features(
             pd.DataFrame: Input DataFrame with an extra column for every added feature.
         """
 
+        if latency_config is None:
```

Comment:
Just by reading the docstring its not totally clear when exactly and which features will not be included. I assume by default only the timedriven features based on the historic load are not included in some cases. Perhaps we can make this even more clear (or perhaps I just have to read it better).